### PR TITLE
Adapted to scp 096, version 10.1 update

### PR DESCRIPTION
**Pull request by ismail0234**

### CODE REVIEW

**Comment by ismail0234** [comment]
Pull-request number #292 continues.

**Comment by ismail0234** [comment]
Ooops Wrong

File: Exiled.Events/Patches/Events/Scp096/AddingTarget.cs [hunk 1]
```diff
@@ -43,15 +48,16 @@ private static bool Prefix(Scp096 __instance, GameObject target)
                 return true;
             }
 
-            AddingTargetEventArgs ev = new AddingTargetEventArgs(scp096, targetPlayer, 200, __instance.EnrageTimePerReset);
+            AddingTargetEventArgs ev = new AddingTargetEventArgs(scp096, targetPlayer, 70, __instance.EnrageTimePerReset);
             Exiled.Events.Handlers.Scp096.OnAddingTarget(ev);
-
+
```
Comment (iRebbok) [inline Exiled.Events/Patches/Events/Scp096/AddingTarget.cs:53]:
```suggestion

```

**Review by iRebbok — CHANGES_REQUESTED** [review]

**Comment by iRebbok** [comment]
@ismail0234 now fix errors you got, those are StyleCop errors

**Comment by ismail0234** [comment]
Is it really necessary to do this?

**Comment by iRebbok** [comment]
Of course

**Comment by ismail0234** [comment]
It's not possible for me to do this now

**Pull request merged**